apps/src/music/blockly/advancedProcedures.ts

```diff
@@ -0,0 +1,9 @@
+import musicI18n from '../locale';
```

Comment:
Not much happens here yet, but I still opted for a separate file so that there is a space ready for building out further customizations.